src/i18n/MControlCenter_tr.ts

```diff
@@ -0,0 +1,230 @@
+    </message>
+    <message>
+        <source>WebCam</source>
+        <translation>Web Kamerası</translation>
+    </message>
+    <message>
+        <source>USB Power Share</source>
+        <translation>USB Güç Paylaşımı</translation>
+    </message>
+    <message>
+        <source>Debug</source>
+        <translation>Hata ayıkla</translation>
+    </message>
+    <message>
+        <source>About</source>
+        <translation>Hakkında</translation>
+    </message>
+    <message>
+        <source>Version:</source>
+        <translation>Versiyon:</translation>
+    </message>
+    <message>
+        <source>GitHub:</source>
+        <translation>GitHub:</translation>
+    </message>
+    <message>
+        <source>Charging status:</source>
+        <translation>Pil durumu:</translation>
+    </message>
+    <message>
+        <source>Charging</source>
+        <translation>Doluyor</translation>
+    </message>
+    <message>
+        <source>Discharging</source>
+        <translation>Boşalıyor</translation>
+    </message>
+    <message>
+        <source>Not charging</source>
+        <translation>Dolmuyor</translation>
+    </message>
+    <message>
+        <source>Unknown</source>
+        <translation>Bilinmiyor</translation>
+    </message>
+    <message>
+        <source>Bug tracker:</source>
+        <translatorcomment>not sure on this one</translatorcomment>
+        <translation>Hata takibi:</translation>
+    </message>
+    <message>
+        <source>An application that allows you to change the settings of MSI laptops running Linux</source>
+        <translation>Linux çalıştıran MSI laptopların ayarlarını değiştirmenizi sağlayan bir program</translation>
+    </message>
+    <message>
+        <source>Swap FN and Super buttons</source>
+        <translation>FN ve Super butonlarını değiştir</translation>
+    </message>
+    <message>
+        <source>Fully charged</source>
+        <translation>Tamamen dolu</translation>
+    </message>
+    <message>
+        <source>Fully charged (Discharging)</source>
+        <translation>Tamemen dolu (Boşalıyor)</translation>
+    </message>
+    <message>
+        <source>rpm</source>
+        <translation>rpm</translation>
+    </message>
+    <message>
+        <source>Mode</source>
+        <translation>Mod</translation>
+    </message>
+    <message>
+        <source>High Performance</source>
+        <translation>Yüksek Performans</translation>
+    </message>
+    <message>
+        <source>A mode that brings a balance between performance and battery live</source>
+        <translation>Performans ile pil ömrünü dengeleyen bir mod</translation>
+    </message>
+    <message>
+        <source>Silent</source>
+        <translation>Sessiz</translation>
+    </message>
+    <message>
+        <source>Super Battery</source>
+        <translatorcomment>I don&apos;t know what this is</translatorcomment>
+        <translation>Super Battery</translation>
```

Comment:
```suggestion
        <translation type="unfinished">Super Battery</translation>
```

In this case, it's better to leave unfinished